Kuroba/app/src/main/java/com/github/k1rakishou/chan/core/usecase/KurobaSettingsImportUseCase.kt

```diff
@@ -478,7 +478,7 @@ class KurobaSettingsImportUseCase(
             return@jsonObject
           }
 
-          if (action == FilterAction.WATCH.id) {
+          if (action == FilterAction.WATCH.id || action == FilterAction.AVOID_WATCH.id) {
```

Comment:
By the way, you can just remove this because this use case is used for importing data from the other Kuroba application and since it never supported `AVOID_WATCH` filters there is no point to have this additional check.